packages/fxa-auth-server/config/index.ts

```diff
@@ -794,6 +794,32 @@ const conf = convict({
       env: 'SUBSCRIPTIONS_ENABLED',
       default: false,
     },
+    paypalNvpSigCredentials: {
+      sandbox: {
+        doc: 'PayPal Sandbox mode',
+        format: Boolean,
+        env: 'PAYPAL_SANDBOX',
+        default: true,
+      },
+      user: {
+        doc: 'PayPal NVP API User name',
+        format: String,
+        default: 'user',
+        env: 'PAYPAL_NVP_USER',
+      },
+      pwd: {
+        doc: 'PayPal NVP API password',
+        format: String,
+        default: 'user',
```

Comment:
heh not sure if you care about the actual value here, but this looks like an artifact of copy & paste.  Same with `signature` below.

Reply:
Oh, I did put that in on purpose just so there's some value so that it runs even if its not used.